introduction.md

```diff
@@ -1,2 +1,14 @@
 Hello, Dojo, My name is Steve.
 My favorite code school path is javascript!
+
+Deadly Skills
+=============
+* git
+* javascript
+* jquery
+* coffeescript
+* bootstrap
+* web design
+* sass
+* mobile
+* css
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?